internal/namespaces/instance/v1/instance_cli_test.go

```diff
@@ -4,12 +4,19 @@ import (
 	"testing"
 
 	"github.com/scaleway/scaleway-cli/internal/core"
+	"github.com/scaleway/scaleway-sdk-go/api/instance/v1"
+	"github.com/stretchr/testify/assert"
 )
 
+func init() {
+	if !core.UpdateGolden {
+		instance.RetryInterval = 0
```

Comment:
why not do it next to `var UpdateGolden = os.Getenv("UPDATE_GOLDEN") == "true"` ?

Reply:
We scope it per namespace in the SDK, I just put it there to be consistent